kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -203,19 +210,8 @@ var TextView = BaseView.extend({
         return this;
     },
 
-    text_changed: function() {
-        this.model.set(this.text_attr, this.$(".rating-text-feedback")[0].value);
-    },
-
-    toggle_disabled: function(val) {
-        if( typeof val === "undefined" ) {
-            val = !this.is_disabled();
-        }
-        this.$(".rating-text-feedback").attr("disabled", val);
-        return this;
-    },
-
-    is_disabled: function() {
-        return this.$(".rating-text-feedback").attr("disabled") === "disabled";
-    }
+    text_changed: _.throttle(function() {

```

Comment:
The only place we might lose data then is if someone closes their browser window, but apart from preventing browser window closures until we have no 'dirty' models, there's not much we can do about that.